plugins/CorePluginsAdmin/vue/src/PluginSettings/PluginSetting.vue

```diff
@@ -0,0 +1,75 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <div>
+    <FormField
+      :model-value="modelValue"
+      @update:model-value="changeValue($event)"
+      :form-field="{
+        ...setting,
+        condition: conditionFunction,
+      }"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { IScope } from 'angular';
+import { Matomo } from 'CoreHome';
+import FormField from '../FormField/FormField.vue';
+
+// TODO: have to use angularjs here until there's an expression evaluating alternative
+let conditionScope: IScope;
+
+export default defineComponent({
+  props: {
+    pluginName: String,
+    setting: Object,
+    modelValue: Object,
```

Comment:
Opening the settings page logs a couple of warnings in the browser console:
![image](https://user-images.githubusercontent.com/1579355/146748971-1437ffb6-28a5-4dcd-b67b-133eabf72bb2.png)
Not sure if that's something that needs to be fixed or not.

Reply:
👍 worth fixing